fronts-client/src/components/feed/RecipeSearchContainer.tsx

```diff
@@ -109,22 +134,33 @@ export const RecipeSearchContainer = ({ rightHandContainer }: Props) => {
         </TextInputContainer>
         <ClipboardHeader />
       </InputContainer>
-      <RadioGroup>
-        <RadioButton
-          checked={selectedOption === FeedType.recipes}
-          onChange={() => setSelectedOption(FeedType.recipes)}
-          label="Recipes"
-          inline
-          name="recipeFeed"
-        />
-        <RadioButton
-          checked={selectedOption === FeedType.chefs}
-          onChange={() => setSelectedOption(FeedType.chefs)}
-          label="Chefs"
-          inline
-          name="chefFeed"
-        />
-      </RadioGroup>
+      <TopOptions>
+        <RadioGroup>
+          <RadioButton
+            checked={selectedOption === FeedType.recipes}
+            onChange={() => setSelectedOption(FeedType.recipes)}
+            label="Recipes"
+            inline
+            name="recipeFeed"
+          />
+          <RadioButton
+            checked={selectedOption === FeedType.chefs}
+            onChange={() => setSelectedOption(FeedType.chefs)}
+            label="Chefs"
+            inline
+            name="chefFeed"
+          />
+        </RadioGroup>
+        {selectedOption === FeedType.chefs && chefsPagination && hasPages && (
```

Comment:
Do we not also need pagination for recipes? (Is there a search for recipes yet? I can’t remember)

Reply:
Sorry, meant to check this before I finished the review! I see that we don’t have recipe search yet, so no pagination is needed: ignore me.